False positive "differs only by captalization" on alignment fields in struct

**Describe the bug**

A struct with more than one `_` field (e.g. for alignment) is reported to have fields differing only by capitalization.

**To Reproduce**

```shell
% cat a.go 
// package main is a package
package main

type siginfoChild struct {
	signo    int32
	errno    int32
	exitCode int32
	_        int32

	pid    int32
	uid    uint32
	status int32

	_ [25]uint32 // Pad to 128 bytes
}

% go run github.com/mgechev/revive@HEAD -config config.toml a.go
a.go:14:2: Field '_' differs only by capitalization to other field in the struct type siginfoChild
```

```toml
enableAllRules = true
```

**Expected behavior**

A struct may have more than one `_` field.

**Desktop (please complete the following information):**

`go version go1.25.1 darwin/arm64`

--- a/rule/confusing_naming.go
+++ b/rule/confusing_naming.go
@@ -171,7 +171,7 @@ func checkStructFields(fields *ast.FieldList, structName string, w *lintConfusin
 			if id.Name == "_" {
 				continue
 			}
-			
+
 			normName := strings.ToUpper(id.Name)
 			if bl[normName] {
 				w.onFailure(lint.Failure{

--- a/rule/confusing_naming.go
+++ b/rule/confusing_naming.go
@@ -171,6 +171,7 @@ func checkStructFields(fields *ast.FieldList, structName string, w *lintConfusin
 			if id.Name == "_" {
 				continue
 			}
+			
 			normName := strings.ToUpper(id.Name)
 			if bl[normName] {
 				w.onFailure(lint.Failure{

--- a/rule/confusing_naming.go
+++ b/rule/confusing_naming.go
@@ -167,7 +167,7 @@ func checkStructFields(fields *ast.FieldList, structName string, w *lintConfusin
 	bl := make(map[string]bool, len(fields.List))
 	for _, f := range fields.List {
 		for _, id := range f.Names {
-			// Skip blank identifiers - they can appear multiple times for padding/alignment
+			// Skip blank identifiers
 			if id.Name == "_" {
 				continue
 			}

--- a/rule/confusing_naming.go
+++ b/rule/confusing_naming.go
@@ -167,6 +167,10 @@ func checkStructFields(fields *ast.FieldList, structName string, w *lintConfusin
 	bl := make(map[string]bool, len(fields.List))
 	for _, f := range fields.List {
 		for _, id := range f.Names {
+			// Skip blank identifiers - they can appear multiple times for padding/alignment
+			if id.Name == "_" {
+				continue
+			}
 			normName := strings.ToUpper(id.Name)
 			if bl[normName] {
 				w.onFailure(lint.Failure{

--- a/testdata/confusing_naming1.go
+++ b/testdata/confusing_naming1.go
@@ -93,5 +93,5 @@ type siginfoChild struct {
 	uid    uint32
 	status int32
 
-	_ [25]uint32 // Pad to 128 bytes
+	_ [25]uint32
 }

--- a/testdata/confusing_naming1.go
+++ b/testdata/confusing_naming1.go
@@ -81,3 +81,17 @@ type b[T any] struct{}
 
 func (x *b[T]) method() {
 }
+
+// Multiple blank identifiers should be allowed (for padding/alignment)
+type siginfoChild struct {
+	signo    int32
+	errno    int32
+	exitCode int32
+	_        int32
+
+	pid    int32
+	uid    uint32
+	status int32
+
+	_ [25]uint32 // Pad to 128 bytes
+}

